definitions/npm/storybook-router_v0.x.x/flow_v0.25.x-/storybook-router_v0.x.x.js

```diff
@@ -0,0 +1,35 @@
+type LocationShape = {
+  pathname?: string,
+  search?: string,
+  hash?: string,
+  state?: any
+};
+
+type GetUserConfirmation = (
+  message: string,
+  callback: (confirmed: boolean) => void
+) => void;
+
+type Renderable = React$Element<*>;
```

Comment:
can't we import this from React ?

import type { Element } from 'react'?

or is it better to use private def?

Reply:
Yeah I think that's safe. Would it be `type Renderable = Element<*>` or `type Renderable = Element`?